optimize_epanet.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+from scipy.optimize import differential_evolution
+from epanet_end_error import run_epanet_and_calculate_error
+
+
+junctions_to_adjust = [' SO/ZD', ' SW/SO', ' SW/RO', ' SD/GO1', ' SW/AN', ' HP7']
```

Comment:
[Informacja] Takie stałe globalne to zazwyczaj się pisze samymi dużymi literami
(`JUNCTIONS = ...`)